docker-compose.yml

```diff
@@ -80,6 +84,17 @@ services:
       - POSTGRES_PASSWORD=secret
       - POSTGRES_DB=hydra
 
+  jaeger:
+    image: jaegertracing/all-in-one:latest
```

Comment:
The problem with `:latest` tags is that, while they do fetch the latest tag, they don't fetch those the next time you're running `:latest`. This can lead to problems where machine A uses version 1.0 and machine B uses 2.0 although both have the same tag. This can then lead to a ton of confusion and issues. It's much better (and safer) to explicitly define the version.

That being said, I don't think jaeger belongs in this docker-compose file. Instead I'd love to see either a docker command (e.g. `docker run --network --rm ..... jeagertracing/all-in-one-latest`) that you can just run "on top" of this docker-compose set up, or alternatively add this to one of the [ory/examples](https://github.com/ory/examples). If it's the latter, please coordinate with me as I'm currently working on that repository and changing some substantial things.